frontend/src/components/student/studentCommunicationList.tsx

```diff
@@ -0,0 +1,117 @@
+import useSWR from "swr";
+import apiPaths from "../../properties/apiPaths";
+import { getParamsFromQueryUrl, getQueryUrlFromParams } from "../../api/calls/baseCalls";
+import { extractIdFromStudentUrl, getStudentOnUrl } from "../../api/calls/studentCalls";
+import { ICommunication } from "../../api/entities/CommunicationEntity";
+import { getAllCommunicationFromPage } from "../../api/calls/communicationCalls";
+import CommunicationListItem from "../communication/communicationListItem";
+import { Button, Col, Row } from "react-bootstrap";
+import applicationPaths from "../../properties/applicationPaths";
+import { useRouter } from "next/router";
+import { Accordion, Container } from "react-bootstrap";
+import AccordionItem from "react-bootstrap/AccordionItem";
+
+/**
+ * Properties needed for [StudentCommunicationList].
+ */
+export interface CommunicationListProps {
+    studentUrl: string | undefined;
```

Comment:
Why do you allow undefined? maybe use `?:` ?

Reply:
The studentUrl can be undefined when refreshing the communications page

Reply:
Nevermind using an emptyUser fixes this